### PR TITLE
polish etcd_common role

### PR DESCRIPTION
**Pull request by ingvagabund**
- remove unused etcdctl.sh file
- update etcd_common README to capture the current offer of the role

### CODE REVIEW

**Comment by ingvagabund** [comment]
@sdodson @ashcrow Yo, are there any other nits you can recall?

**Comment by ingvagabund** [comment]
aos-ci-test

File: roles/etcd_common/README.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-Role Variables
---------------
+Or etcdctl.yml playbook for installation of `etcdctl` aliases on a node. E.g.
 
-TODO
+```yaml
+- include_role:
+    name: etcd_common
+    tasks_from: etcdctl
+```
```
Comment (sdodson) [inline roles/etcd_common/README.md]: Get rid of this in favor of the example playbook below? Just seems like duplication.

Comment (ingvagabund) [inline roles/etcd_common/README.md]: Done

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
@sdodson PTAL

**Comment by sdodson** [comment]
[merge]

**Comment by sdodson** [comment]
flake https://github.com/openshift/origin/issues/14122
[merge]

**Pull request merged**